compliance_checker/ioos.py

```diff
@@ -590,38 +590,38 @@ def check_contributor_role_and_vocabulary(self, ds):
         role_val = False
         vocb_val = False
 
-        role_msg = "contributor_role should be from NERC or GEOIDE"
-        vocb_msg = "contributor_role_vocabulary should be one of NERC or GEOIDE"
+        role_msg = "contributor_role should be from NERC or NOAA-NCEI"
+        vocb_msg = "contributor_role_vocabulary should be one of NERC or NOAA-NCEI"
 
         if role:
             if role in [
-                "author",
-                "coAuthor",
-                "collaborator",
-                "contributor",
-                "custodian",
-                "distributor",
-                "editor",
-                "funder",
-                "mediator",
-                "originator",
-                "owner",
-                "pointOfContact",
-                "principalInvestigator",
-                "processor",
-                "publisher",
-                "resourceProvider",
-                "rightsHolder",
-                "sponsor",
-                "stakeholder",
-                "user"
+                "author",                # both
```

Comment:
Is it necessary to have `# both` after each line here?  I think this could be eliminated without losing meaning.